calicoctl/commands/node/diags.go

```diff
@@ -134,6 +136,9 @@ func runDiags(logDir string) {
 		fmt.Printf("No logs found in %s; skipping log copying", logDir)
 	}
 
+	// Try to copy logs from containers for hosted installs.
+	getHostedLogs(tmpLogDir)
```

Comment:
a nit: this isn't specific to "self-hosted" calico installs - this is just "get stdout logs from any calico containers"

So, basically a change to the comment and function name (e.g. `getNodeContainerLogs`)